REQUIRE

```diff
@@ -1,3 +1,3 @@
-julia 0.5
-MathProgBase 0.5 0.8
-JuMP 0.15
+julia 0.6
+MathProgBase 0.7
```

Comment:
Why remove the upper bound on MPB? It's there to avoid broken packages if there are breaking changes in MPB.